### DISCUSSION TITLE
Assignment 1 , Module 2

### DISCUSSION THREAD

**Discussion by Achidago** [SERVIR-WA/GALUP#36] [General]
The "osm2020_allrds_THLD.shp" layer has to be reprojected to "EPSG:32630 – WGS 84 / UTM zone 30N" to be able to use in "Density of line features" tool (script).

**Comment by chjch** [comment]
@Achidago  Thanks Achidago. This is not our intention. We will upload the projected roads to the repository.

**Reply by Achidago** [reply]
Well noted. Thanks
